src/node.cc

```diff
@@ -3717,7 +3711,7 @@ static void ParseArgs(int* argc,
         exit(9);
       }
       args_consumed += 1;
-      local_preload_modules[preload_module_count++] = module;
+      preload_modules.push_back(module);
```

Comment:
How? This is only inserting one element, I don't know how many `-r` options were passed.